happi/containers.py

```diff
@@ -297,3 +297,33 @@ class Motor(Device):
     device_class.default = 'pcdsdevices.device_types.Motor'
     system = copy(Device.system)
     system.default = 'motion'
+
+
+class AreaDetector(Device):
+    """
+    A Generic EpicsCamera
+    """
+    device_class = copy(Device.device_class)
+    device_class.default = 'pcdsdevices.device_types.PCDSDetector'
+    system = copy(Device.system)
+    system.default = 'camera'
+
+
+class Acromag(Device):
+    """
+    A Generic class for Acromag
+    """
+    device_class = copy(Device.device_class)
+    device_class.default = 'pcdsdevices.device_types.Acromag'
```

Comment:
Does this class exist?

Reply:
It does with this PR: https://github.com/pcdshub/pcdsdevices/pull/329